src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/DefaultRazorHoverInfoService.cs

```diff
@@ -125,7 +125,10 @@ public override HoverModel GetHoverInfo(RazorCodeDocument codeDocument, SourceLo
                     Debug.Assert(binding.Descriptors.Any());
                     var tagHelperAttributes = _tagHelperFactsService.GetBoundTagHelperAttributes(tagHelperDocumentContext, selectedAttributeName, binding);
 
-                    var attribute = attributes.Single(a => a.Span.IntersectsWith(location.AbsoluteIndex));
+                    // Grab the first attribute that we find that intersects with this location. That way if there are multiple attributes side-by-side aka hovering over:
+                    //      <input checked| minimized />
+                    // Then we take the left most attribute (attributes are returned in source order).
+                    var attribute = attributes.First(a => a.Span.IntersectsWith(location.AbsoluteIndex));
```

Comment:
Is there any value in having info for both attributes being hovered over?

Ie. 

```
INFO 1
_________________

INFO 2
```

Reply:
Not entirely because in this case the "edge" is only an "edge" because the second attribute encompasses the prefixed whitespace